processing/src/main/java/org/apache/carbondata/processing/merger/CarbonCompactionExecutor.java

```diff
@@ -109,8 +110,8 @@ public CarbonCompactionExecutor(Map<String, TaskBlockInfo> segmentMapping,
    * Map(UNSORTED) = List of Iterators which yield sorted data
    * Map(Sorted) = List of Iterators which yield sorted data
    */
-  public Map<String, List<RawResultIterator>> processTableBlocks(Configuration configuration)
-      throws QueryExecutionException, IOException {
+  public Map<String, List<RawResultIterator>> processTableBlocks(Configuration configuration,
+      Expression andFilterExpr) throws QueryExecutionException, IOException {
```

Comment:
Add comment about andFilterExpr above the method

Reply:
Better just name as `filterExpr`

Reply:
Done.